### PR TITLE
Update FAQ section in User Guide

### PR DESCRIPTION
**Pull request by WeeeHung**
- added some frequently asked questions and information about CLI applications that are missed out in the user guide.
- included unique ids to headers to allow linking. Sections can now be hyperlinked from other parts of the user guide.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -249,23 +249,48 @@ _Details coming soon ..._
...
+**Q**: What operating systems does HealthSync support?<br>
+**A**: HealthSync is a cross-platform application and supports Windows, macOS, and Linux. Simply follow the
+[installation guide](#quick-start) and download the jar file to use the application.
+
+**Q**: What are the advantages of using a CLI interface in HealthSync over traditional GUI apps?<br>
+**A**: HealthSync's CLI interface offers faster data entry and navigation for users who are comfortable with typing.
+It streamlines tasks and provides a more efficient way to manage patient details.
+
+**Q**: Can multiple users access Healthysync simultaneously, and how do I set up user accounts?<br>
+**A**: Healthysync is currently an application for a single user. Hence, it cannot be accessed simultaneously
```
Comment (hyc17003) [inline docs/UserGuide.md:263]:
```suggestion
**A**: HealthSync is currently an application for a single user. Hence, it cannot be accessed simultaneously
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -249,23 +249,48 @@ _Details coming soon ..._
...
+by different users.
+
+**Q**: How do I import patient data from external sources into Healthysync?<br>
```
Comment (hyc17003) [inline docs/UserGuide.md:266]:
```suggestion
**Q**: How do I import patient data from external sources into HealthSync?<br>
```

**Comment by hyc17003** [comment]
Some typos to amend, other than that, FAQ section is concise and informative. LGTM.

**Comment by kanna-1** [comment]
Other than the aforementioned changes suggested by @hyc17003, LGTM.

File: docs/UserGuide.md (hunk 1 above)
Comment (kanna-1) [reply docs/UserGuide.md:263]: I agree with the change, committing the suggestion.

**Pull request merged**